tests/contrib/logging/test_logging.py

```diff
@@ -24,18 +24,23 @@ def current_span(tracer=None):
     return tracer.current_span()
 
 
-def capture_function_log(func, fmt=DEFAULT_FORMAT):
+def capture_function_log(func, fmt=DEFAULT_FORMAT, logger_arg=None):
+    if logger_arg is not None:
+        logger_to_capture = logger_arg
```

Comment:
A more explicit variable name

```suggestion
def capture_function_log(func, fmt=DEFAULT_FORMAT, logger_override=None):
    if logger_arg is not None:
        logger_to_capture = logger_override
```